deno.importmap

```diff
@@ -1,8 +1,10 @@
 {
   "scopes": {
     "./": {
+      "dayjs/esm": "https://ga.jspm.io/npm:dayjs@1.10.7/esm/index.js",
```

Comment:
Minor thing, no change needed though. 
We can have it directly like 
```
"dayjs": "https://ga.jspm.io/npm:dayjs@1.10.7/esm/index.js",
```

Reply:
the auto generated one doesn't map to esm version so I intentionally used this.